vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -652,7 +654,7 @@ base_learner* ccb_explore_adf_setup(options_i& options, vw& all)
     options.add_and_parse(new_options);
   }
 
-  if (!options.was_supplied("cb_sample"))
+  if (!options.was_supplied("cb_sample") && !slate)
```

Comment:
If `cb`_sample was supplied, then `slate` is ignored. This behavior is confusing.
The 2 options are not compatible, so why not catching/throwing from here?

Reply:
I've updated with a throw